### PR TITLE
Roadmap v0.9

### PR DESCRIPTION
**Pull request by cheina97**
# Description

This PR adds the roadmap for v0.9

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Roadmap
 
-Planned features for the next release (v0.8, expected Apr 2023) include the following:
+Planned features for the next release (v0.9, expected early June 2023) include the following:
```
Comment (frisso) [inline ROADMAP.md:3]:
```suggestion
Planned features for the next release (v0.9, expected June 2023) include the following:
```

File: ROADMAP.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
-* Support for external network providers (e.g. Cilium, Submariner, etc.). Or no network provider at all.
-* Improved resilience and availability in cluster disconnection scenarios.
+* Make it possible to target a remote cluster's nodes with Multiple Virtual Nodes (check [this](https://github.com/liqotech/liqo/issues/1768) and [this](https://github.com/liqotech/liqo/issues/1766) issues).
```
Comment (frisso) [inline ROADMAP.md:5]:
```suggestion
* Enable to summarize the resources of the remote cluster with more than one Virtual Node (check [this](https://github.com/liqotech/liqo/issues/1768) and [this](https://github.com/liqotech/liqo/issues/1766) issues).
```

**Review by aleoli — APPROVED** [review]

**Comment by cheina97** [comment]
/ merge

**Pull request merged**